docs/source/intro/installation.rst

```diff
@@ -40,6 +40,12 @@ To assume the role of ``postgres`` run as root:
 
    $ psql
 
+.. tip::
+
+   If you see an error message like ``psql: FATAL:  role "<role_name>" does not exist``, probably you don't have any role created yet.
+   You can run ``psql -l`` to check the exist roles and specify the correct one with the ``-U`` option.
```

Comment:
You are using the term "role" here, but further down the language is to create a "user". I wonder if this is going to be obvious to users that they are equivalent
```suggestion
   If you see an error message like ``psql: FATAL:  role "<role_name>" does not exist``, probably you have not created any roles yet.
   You can run ``psql -l`` to check the existing roles and specify the correct one with the ``-U`` option.
```